stack.yaml

```diff
@@ -1,5 +1,5 @@
 resolver: nightly-2022-08-04
 
 extra-deps:
-  - github: uhbif19/colourista
-    commit: 8148a0446bf61814f79d6a0c497007fde72b31eb
+  - github: kowainik/colourista
+    commit: c7b06816ab85c27439fa7afbe6d5eed0506dd884
```

Comment:
I don't think we need to depend on a GitHub commit and we can use the version from Hackage instead:

```yaml
- colourista-0.1.0.2
```